drivers/hud_apr/app/models/hud_apr.rb

```diff
@@ -0,0 +1,20 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudApr
+  def self.current_generator(report:)
```

Comment:
Ah, I see, this is hard-coded here rather than reflecting the active version as defined in the controller.  I guess that's ok then.  Thanks!